specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -1080,10 +1127,10 @@
         "fragmentsPerTsSegment": {
           "format": "int32",
           "type": "integer",
-          "description": "The amount of fragments per HTTP Live Streaming (HLS) segment."
+          "description": "The number of fragments in an HTTP Live Streaming (HLS) segment in the output of the live event."
```

Comment:
This does not impact HLS CMAF, only the legacy HLS v3 with TS segements.  We should make that clear here.  We have a gap still in that we do not provide a method for fragments per CMAF segment when using HLS.

Reply:
Shall we re-phrase to say: "The number of fragments in an HTTP Live Streaming (HLS) TS segment in the output of the live event. This setting does not impact HLS CMAF or DASH outputs."